src/components/ChatLog.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+
+const ChatLog = ({entries, updateLikes}) => {
+    return (
+        <div className="chat-log">
+            {entries.map((chatEntry) => (
+                <ChatEntry
+                    key={chatEntry.id}
+                    sender={chatEntry.sender}
+                    body={chatEntry.body}
+                    timeStamp={chatEntry.timeStamp}
+                    liked={chatEntry.liked}
+                    updateLikes={() => updateLikes(chatEntry.id)}
+                />
+            ))}
+        </div>
+    );
+};
+ChatLog.propTypes={
+    entries:PropTypes.arrayOf(
+        PropTypes.shape({
```

Comment:
👍 Nice use of `shape` to describe the data the values within the array should have.